Doc/library/os.rst

```diff
@@ -1108,6 +1108,17 @@ as internal buffering of data.
    .. versionchanged:: 3.12
       Added support for pipes on Windows.
 
+
+.. function:: grantpt(fd, /)
+
+   Grant access to the slave pseudo-terminal device associated with the
+   master pseudo-terminal device to which the file descriptor *fd* refers.
+
+   Performs a :c:func:`grantpt` system call.
```

Comment:
While I have used the term "system call" here and in other places for the other three functions, these are under section 3 ("library functions") as opposed to section 2 ("system calls") of Linux and most BSD manpages, except `posix_openpt()` is under section 2 of FreeBSD. Is there a better term that we should use?